Update search.md (#6924)
DataDog/documentation: content/en/tracing/app_analytics/search.md

{{< img src="tracing/app_analytics/search/facets_demo.png" alt="Facets demo"  style="width:80%;">}}

> ### Quantitative facets: Measures

Reviewer: ```suggestion
### Quantitative facets: measures
```

> **Use measures when you need:** 

Reviewer: ```suggestion
**Use measures when you need to:** 
```

> * To aggregate values from multiple Traces. For instance, create a measure on the number of rows in Cassandra and view the P95 or top-most referrers per sum of file size requested.

Reviewer: ```suggestion
* Aggregate values from multiple traces. For example, create a measure on the number of rows in Cassandra and view the P95 or top-most referrers per sum of file size requested.
```

> * To numerically compute the highest latency services for shopping cart values over $1000

Reviewer: ```suggestion
* Numerically compute the highest latency services for shopping cart values over $1000.
```

> *To filter continuous values. For instance, size in bytes of each payload chunk of a video stream	

Reviewer: ```suggestion
* Filter continuous values. For example, the size in bytes of each payload chunk of a video stream.	
```

**Types**

Measures come with either a (long) integer or double value, for equivalent capabilities.

**Units**

> Measures support units (time in seconds or size in bytes) for easier handling of orders of magnitude at query time and display time. Unit is a property of the measure itself, not of the field. For example, consider a duration measure in nanoseconds: you have a span tag from `service:A` where `duration:1000` stands for `1000 milliseconds`, and other span tags from `service:B` where `duration:500` stands for `500 microseconds`:

Reviewer: ```suggestion
Measures support units (time in seconds or size in bytes) for handling of orders of magnitude at query time and display time. Unit is a property of the measure itself, not of the field. For example, consider a duration measure in nanoseconds: you have a span tag from `service:A` where `duration:1000` stands for `1000 milliseconds`, and another span tags from `service:B` where `duration:500` stands for `500 microseconds`:
```

Scale duration into nanoseconds for all span tags flowing in with the arithmetic processor. Use a `*1000000` multiplier on span tags from `service:A`, and a `*1000` multiplier on span tags from `service:B`.

> Use `duration:>20ms` (see search syntax for reference) to consistently query span tags from both services at once, and see an aggregated result of max 1 min.

Reviewer: ```suggestion
Use `duration:>20ms` (see search syntax for reference) to consistently query span tags from both services at once, and see an aggregated result of max one minute.
```
